### PR TITLE
Added Accordion code page

### PR DESCRIPTION
**Pull request by raquelarrojo**
Closes [#981](https://github.com/dxc-technology/halstack-react/issues/981).

### CODE REVIEW

File: website/screens/components/accordion/usage/examples/assistiveText.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  return (
+    <DxcInset space="large">
+      <DxcAccordion
+        label="Collapsed accordion"
+        assistiveText="Additional information"
+        isExpanded={isExpanded}
+        onChange={onChange}
+        padding="medium"
```
Comment (jsuarezgonz) [inline website/screens/components/accordion/usage/examples/assistiveText.ts:17]: Please remove this padding and change the DxcInset space.

File: website/screens/components/accordion/usage/examples/behaviorAndInteraction.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  };
+
+  return (
+    <DxcInset space="large">
+      <DxcStack gutter="large">
+        <DxcAccordion
+          label="Collapsed"
+          isExpanded={isExpandedCollapsed}
+          onChange={onChangeCollapsed}
+          padding="medium"
```
Comment (jsuarezgonz) [inline website/screens/components/accordion/usage/examples/behaviorAndInteraction.ts:26]: Please remove this padding and change the DxcInset space.

**Review by jsuarezgonz — CHANGES_REQUESTED** [review]

File: website/screens/components/accordion/code/examples/uncontrolledAccordion.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import { useState } from "react";
+
+const code = `() => {
+  return (
+    <DxcInset space="large">
+      <DxcAccordion
+        label="Uncontrolled"
+        assistiveText="Additional information"
+        defaultIsExpanded
+        padding="medium"
```
[the remark this answers is not in the document]
Comment (raquelarrojo) [reply website/screens/components/accordion/code/examples/uncontrolledAccordion.ts]: Done!

File: website/screens/components/accordion/code/examples/controlledAccordion.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  return (
+    <DxcInset space="large">
+      <DxcAccordion
+        label="Controlled"
+        isExpanded={isExpanded}
+        assistiveText="Additional information"
+        onChange={onChange}
+        padding="medium"
```
[the remark this answers is not in the document]
Comment (raquelarrojo) [reply website/screens/components/accordion/code/examples/controlledAccordion.ts]: Done!

**Review by jsuarezgonz — APPROVED** [review]

**Pull request merged**